src/main/java/business/controllers/RequestController.java

```diff
@@ -547,6 +547,9 @@ public RequestRepresentation claim(
         instance = requestService.getProcessInstance(id);
         RequestRepresentation updatedRequest = new RequestRepresentation();
         requestFormService.transferData(instance, updatedRequest, user.getUser());
+        if (updatedRequest.getAssigneeName() != null) {
+            requestFormService.updateLastAssignee(instance.getId(), updatedRequest.getAssigneeName());
```

Comment:
Updating the last assignee with the assignee name means that when the name of a user is edited, the old name will remain here. I think it is better to store the user id and fetch the name by id.

Reply:
Good point! Now it is updated (with some small refactoring).